EIPS/eip-2315.md

```diff
@@ -317,7 +317,7 @@ Error: at pc=0, op=RETURNSUB: invalid retsub
 
 In this example the RJUMPSUB is on the last byte of code. When the subroutine returns, it should hit the 'virtual stop' _after_ the bytecode, and not exit with error
 
-Bytecode: `0x5c035e5ff` (`RJUMP 3, RETURNSUB, RJUMPSUB -1`)
+Bytecode: `0x5c035e5fff` (`RJUMP 3, RETURNSUB, RJUMPSUB -1`)
```

Comment:
```suggestion
Bytecode: `0x5c035e5f` (`RJUMP 3, RETURNSUB, RJUMPSUB -1`)
```
Isn't this the intention, rather than adding an `ff`? 

> In this example the RJUMPSUB is on the last byte of code.

Reply:
I do not think so:
1. RJUMPSUB requires relative offset and it was clearly declared as -1 in the description = 0xff
 -- there was just one f missing - I assume typo
 
2. RETURNSUB returns PC just behind RJUMPSUB - to position 5 - effectively into the STOP code as 0x0 is expected there - so the 'virtual stop' claim is rather optimization for the clients.

We could improve the sentence: 

When the subroutine returns using RETURNSUB, it should hit the 'virtual stop' _after_ the bytecode, and not exit with error